soql-server-pg/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# Set everything to be logged to the file target/unit-tests.log
```

Comment:
@znep ScalaTest does change the thread name to the name of the test, so it's pretty easy to grep out the name of the test.   In fact maybe that's what IntelliJ is doing.   And when you run a lot of tests together, the log output often gets jumbled anyways, meaning you have to grep no matter where it's going.

Anyways I can't seem to use JVM flags to have log4j redirect its output, except for using src/test/resources.  Maybe I'll just keep a local file there and not check it in.